test/AutoRest.TestServerLowLevel.Tests/dpg-customization.cs

```diff
@@ -24,7 +24,14 @@ public class DpgCustomizationTest : TestServerLowLevelTestBase
         [Test]
         public Task GetHandwrittenModel() => Test(async (host) =>
         {
+            using var diagnosticListener = new ClientDiagnosticListener("dpg_customization_LowLevel", asyncLocal: true);
```

Comment:
Would it be possible to also add a test that shows us not suppressing the nested scope and seeing `scopes.Count == 2`?

Reply:
Test case added.

* [Without suppressing the nested scope](https://github.com/Azure/autorest.csharp/blob/a8d3964566b6dbed5d144665ab3dbd328887d9d3/test/AutoRest.TestServerLowLevel.Tests/dpg-customization.cs#L39)
* [With suppressing the nested scope](https://github.com/Azure/autorest.csharp/blob/a8d3964566b6dbed5d144665ab3dbd328887d9d3/test/AutoRest.TestServerLowLevel.Tests/dpg-customization.cs#L25)